samples/AzCopyCore/AzCopyCore/AzCopyCore.csproj

```diff
@@ -19,18 +21,19 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="1.0.0-alpha-26223-02" />
-    <PackageReference Include="System.Memory" Version="$(CoreFxPackageVersion)" />
-    <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(CoreFxPackageVersion)" />
+    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="1.0.0-alpha-26320-02" />
+    <PackageReference Include="System.Buffers.ReaderWriter" Version="0.1.0-preview2-180320-2" />
```

Comment:
nit: Move this down to the other ItemGroup with the rest of the corefxlab packages

Reply:
will do